gatherer/src/main/java/de/zalando/pgobserver/gatherer/ConfigChecker.java

```diff
@@ -28,15 +29,13 @@ public void run() {
         while (true) {
 
             try {
-                Thread.sleep(CONFIG_CHECK_INTERVAL_SECONDS * 1000);
-
                 Map<Integer, Host> hosts_new = Host.LoadAllHosts(config);
 
                 applyConfigChangesIfAny(hosts_new);
 
-                LOG.error("finished checking new config settings. sleeping for {} s", CONFIG_CHECK_INTERVAL_SECONDS);
-            } catch (InterruptedException ie) {
-                LOG.error("", ie);
+                LOG.info("finished checking new config settings. sleeping for {} s", CONFIG_CHECK_INTERVAL_SECONDS);
```

Comment:
It is not going to sleep here, because you removed `Thread.sleep` from the line 31